packages/e2e-test-app/windows/ReactUWPTestApp/TreeDumpControlViewManager.cs

```diff
@@ -1,365 +0,0 @@
-                if (matchingNode != null)
-                {
-                    dumpRoot = matchingNode;
-                }
-            }
-
-            string dumpText = VisualTreeDumper.DumpTree(dumpRoot, textBlock, additionalProperties, mode);
-
-            if (dumpText != dumpExpectedText)
-            {
-                return await MatchDump(dumpText, GetMasterFile(dumpID), GetOutputFile(dumpID));
-            }
-            return true;
-        }
-
-        private async void dispatcherTimer_Tick(object sender, object e)
-        {
-            m_timer.Stop();
-            if (VisualTreeHelper.GetParent(m_textBlock) != null)
-            {
-                var matchSuccessful = await MatchTreeDumpFromLayoutUpdateAsync(m_dumpID, m_uiaID, m_textBlock, m_additionalProperties, DumpTreeMode.Json, m_dumpExpectedText);
-                if (!matchSuccessful)
-                {
-                    StorageFolder storageFolder = KnownFolders.DocumentsLibrary;
-                    StorageFile outFile = await storageFolder.GetFileAsync(GetOutputFile(m_dumpID));
-                    StorageFile masterFile = await Windows.ApplicationModel.Package.Current.InstalledLocation.GetFileAsync($@"Assets\{GetMasterFile(m_dumpID)}");
-
-                    UpdateResult(false /*matchDump*/ ,
-                        $"Tree dump file does not match master at {masterFile.Path} - See output at {outFile.Path}",
-                        GetInlines(masterFile, outFile, m_textBlock));
-                }
-                else
-                {
-                    UpdateResult(true /*matchDump*/ , "");
-                }
-
-            }
-        }
-
-        private static IList<Inline> GetInlines(StorageFile masterFile, StorageFile outFile, UIElement anchor)
-        {
-            Hyperlink masterLink = new Hyperlink();
-            masterLink.Click += async (_1, _2) => { await Windows.System.Launcher.LaunchFileAsync(masterFile); };
-            masterLink.Inlines.Add(new Run() { Text = "master" });
-            Hyperlink outLink = new Hyperlink();
-            outLink.Click += async (_1, _2) => { await Windows.System.Launcher.LaunchFileAsync(outFile); };
-            outLink.Inlines.Add(new Run() { Text = "output" });
-            List<Inline> inlines = new List<Inline>()
-            {
-                new Run () { Text = "Tree dump " },
-                outLink,
-                new Run () { Text = " does not match " },
-                masterLink
-            };
-
-            #region Diff support - Replace with LaunchUriAsync when we find the VSCode protocol handler Uri for diffing
-            string code_cmd = Environment.ExpandEnvironmentVariables(@"%UserProfile%\AppData\Local\Programs\Microsoft VS Code\bin\code.cmd");
-            Hyperlink diffLink = new Hyperlink();
-            diffLink.Click += (_1, _2) =>
-            {
-                string commandLine = $"code.cmd --diff \"{masterFile.Path}\" \"{outFile.Path}\"";
```

Comment:
being able to click in the UI to open the difference is kind of a nice feature... can this kind of functionality be preserved?

Reply:
Jest will show colorized diffs in the test runner, including in CI output for each failing test. See the screenshot in description.